### DISCUSSION TITLE
Persistence with logins

### DISCUSSION THREAD

**Discussion by ruelustin5** [capawesome-team/capacitor-firebase#401] [Q&A] [answered]
I don't know if it's normal, but when I refresh the page (F5) after a login, I'm disconnected. FirebaseAuthentication.getCurrentUser() returns a null (while directly after the login, it did return the user) Should i create our own automatic login system? If so, I don't understand how to do it since there doesn't seem to be a function to login with creditentials. Thank you in advance for your help.

**Comment by robingenz** [comment, marked as answer]
You need to call `initializeAuth` with `persistence` set to `indexedDBLocalPersistence`, see [Installation](https://github.com/capawesome-team/capacitor-firebase/blob/de8608707c83b3d01abe8497c45f6caebbfd6720/packages/authentication/docs/firebase-js-sdk.md#installation).
